data/data/openstack/masters/main.tf

```diff
@@ -132,6 +132,14 @@ resource "openstack_compute_instance_v2" "master_conf_0" {
     }
   }
 
+  dynamic "network" {
+    for_each = [for port in openstack_networking_port_v2.master_0_failuredomain : port.id]
```

Comment:
these are the ports we create based on the non-control-plane portTargets in the corresponding failureDomain.